aeppl/joint_logprob.py

```diff
@@ -17,7 +18,7 @@
 from aeppl.utils import rvs_to_value_vars
 
 
-def joint_logprob(
+def factorized_joint_logprob(
```

Comment:
It's possible we could do away with the `vars` argument entirely, since these are available as the keys of `rv_values`

Reply:
That sounds correct.